docs/en/latest/configuration/config-yaml.md

```diff
@@ -0,0 +1,556 @@
+---
+title: Configuration Options
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+This page describes the configuration options defined in the `conf/config-default.yaml`.
+
+## `apisix`
+
+ <table>
```

Comment:
Can we use markdown type of the table? No one wants to edit HTML on a markdown page.

Reply:
Multi-row and multi-column are not supported by docusaurous yet and so I had to use HTML

Reply:
cc @SkyeYoung Is it possible to implement Multi-row/column on Docusaurus?

Reply:
I answered it here https://github.com/apache/apisix-website/issues/1147

Reply:
We can get rid of it if we use the yaml code block. One of the benefits is `consistent render behavior`.